client/src/components/feedbacks/feedback.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import FeedbackSend from './send';
+import FeedbackComment from './comment';
+import '../../style/feedback.css';
+import Api from '../../services/api';
+
+
+class Feedback extends Component {
+  constructor() {
+    super();
+
```

Comment:
We do pass an `id` prop in [restaurant-item.js](https://github.com/Social-projects-Rivne/Lunch_Time/pull/13/files#diff-a0b9117a158eea0df415ad33c0774363), line 56

So, we also need a constructor with `props` like this:
```
  constructor(props) {
    super(props);
```